tensorflow_addons/losses/BUILD

```diff
@@ -28,3 +28,30 @@ py_test(
     ],
     srcs_version = "PY2AND3",
 )
+
+py_library(
+    name = "lifted_py",
```

Comment:
Please put lifted.py in the first py_library `"losses_py"`, and delete the py_library. We can split them in the future if necessary.

Reply:
Done